packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -34,6 +35,10 @@ class CircleAvatar extends StatelessWidget {
   /// color will cause the avatar to animate to the new color.
   final Color backgroundColor;
 
+  /// The background image of the circle. Changing the background
+  /// image will cause the avatar to animate to the new image.
+  final BackgroundImage backgroundImage;
```

Comment:
This should probably be an ImageProvider rather than an entire background image.  The other properties (e.g., alignment, fit, and repeating) don't make sense for clients to control.